src/open_inwoner/accounts/views/cases.py

```diff
@@ -78,7 +78,12 @@ def dispatch(self, request, *args, **kwargs):
             if not fetch_roles_for_case_and_bsn(self.case.url, request.user.bsn):
                 return self.handle_no_permission()
 
-            # check confidentiality level
+            # resolve case-type
+            self.case.zaaktype = fetch_single_case_type(self.case.zaaktype)
+            if not self.case.zaaktype:
+                return self.handle_no_permission()
```

Comment:
I don't know if this is done for readability purposes but we have five times the same return statement.

Reply:
Yep, to keep the decision procedure more readable (without adding extra noisy workarounds) it just returns after every failed condition.